Source/WebCore/rendering/RenderTextControlMultiLine.cpp

```diff
@@ -58,6 +59,11 @@ bool RenderTextControlMultiLine::nodeAtPoint(const HitTestRequest& request, HitT
     if (!RenderTextControl::nodeAtPoint(request, result, locationInContainer, accumulatedOffset, hitTestAction))
         return false;
 
+    const LayoutPoint localPoint(locationInContainer.point() - (accumulatedOffset + location()));
```

Comment:
I think it would be better to implement `isPointInOverflowControl()` like RenderBlock and RenderListBox do and have it call `hitTestOverflowControls()`, just for consistency with those other classes.

Reply:
OK, I moved `isPointInOverflowControl()` from `private` to `protected` and called it.